lib/util/game_setup.rb

```diff
@@ -69,6 +69,7 @@ def get_order(config, setup_validator)
   def get_input(user_interface, script, setup_validator, choices, error_key)
     begin
       input = user_interface.get_input(script).strip
+      user_interface.clear
```

Comment:
I kinda feel like clear should be a private function in the user_interface, "clearing" feels like it belongs specifically to the domain of a CLI user interface.

Reply:
I concur ☝️

Reply:
Calls to clear should now be contained in the UI class.